[2025-07-19T03:10:03.689+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-07-19T03:10:03.810+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: ferry_portfolio_etl.extract_data scheduled__2025-07-19T03:05:00+00:00 [queued]>
[2025-07-19T03:10:03.834+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: ferry_portfolio_etl.extract_data scheduled__2025-07-19T03:05:00+00:00 [queued]>
[2025-07-19T03:10:03.835+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 2
[2025-07-19T03:10:03.865+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): extract_data> on 2025-07-19 03:05:00+00:00
[2025-07-19T03:10:03.907+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61 DeprecationWarning: This process (pid=483) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-07-19T03:10:03.906+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'ferry_portfolio_etl', 'extract_data', 'scheduled__2025-07-19T03:05:00+00:00', '--job-id', '20', '--raw', '--subdir', 'DAGS_FOLDER/ferry_pipeline_dag.py', '--cfg-path', '/tmp/tmpwqknl6el']
[2025-07-19T03:10:03.909+0000] {standard_task_runner.py:63} INFO - Started process 485 to run task
[2025-07-19T03:10:03.910+0000] {standard_task_runner.py:91} INFO - Job 20: Subtask extract_data
[2025-07-19T03:10:03.947+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/settings.py:195 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2025-07-19T03:10:04.119+0000] {task_command.py:426} INFO - Running <TaskInstance: ferry_portfolio_etl.extract_data scheduled__2025-07-19T03:05:00+00:00 [running]> on host 4a4916efbe02
[2025-07-19T03:10:04.819+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='ashok' AIRFLOW_CTX_DAG_ID='ferry_portfolio_etl' AIRFLOW_CTX_TASK_ID='extract_data' AIRFLOW_CTX_EXECUTION_DATE='2025-07-19T03:05:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-07-19T03:05:00+00:00'
[2025-07-19T03:10:04.821+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-07-19T03:10:04.913+0000] {extract_data.py:78} INFO - üöÄ Starting ferry data extraction pipeline...
[2025-07-19T03:11:43.420+0000] {extract_data.py:80} INFO - üìä Retrieved 249958 rows and 4 columns
[2025-07-19T03:11:43.586+0000] {extract_data.py:21} ERROR - ‚ùå Error saving local temp file: [Errno 13] Permission denied: 'artifacts/ferry_raw.csv'
[2025-07-19T03:11:43.588+0000] {extract_data.py:98} ERROR - üö® Failed in extract_data pipeline: CustomException.__init__() missing 1 required positional argument: 'error_details'
[2025-07-19T03:11:43.626+0000] {python.py:237} INFO - Done. Returned value was: None
[2025-07-19T03:11:43.649+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-07-19T03:11:43.924+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=ferry_portfolio_etl, task_id=extract_data, run_id=scheduled__2025-07-19T03:05:00+00:00, execution_date=20250719T030500, start_date=20250719T031003, end_date=20250719T031143
[2025-07-19T03:11:44.178+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2025-07-19T03:11:44.462+0000] {taskinstance.py:3498} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-07-19T03:11:44.474+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
